builder/bundles.go

```diff
@@ -645,9 +669,9 @@ func installBundleToFull(packagerCmd []string, baseDir string, bundle *bundle, d
 				if e == nil {
 					break
 				}
+				log.Debug(log.Mixer, "RPM Extraction attempt %d failed. Maximum of %d attempts.\n", i+1, extractRetries)
```

Comment:
This could be a `log.Warning()`

Reply:
as extract rpm is unknown to user, it will only be printed in debug mode. if we used log.warning, user will get the output in stdout as well